### PR TITLE
Consistency 2023-02-16

### PR DESCRIPTION
**Pull request by Simran-B**
Fix typos and formatting.
Kudos to @jsoref for his work that enabled me to find a lot of misspelled words.

Upstream: https://github.com/arangodb/arangodb/pull/17471 (devel/3.11 only)

### CODE REVIEW

File: 3.10/foxx-reference-modules-queues.md [hunk 1]
```diff
@@ -294,9 +294,9 @@ queue.push(
 
 Creates a proxy object representing a job with the given job id.
 
-The job will be looked up in the specified queue in the current database.
+The job is looked up in the specified queue in the current database.
 
-Returns the job for the given jobId. Properties of the job object will be
+Returns the job for the given jobId. Properties of the job object are
 fetched whenever they are referenced and can not be modified.
```
Comment (nerpaula) [inline 3.10/foxx-reference-modules-queues.md:300]:
```suggestion
fetched whenever they are referenced and cannot be modified.
```

File: 3.10/foxx-reference-modules-queues.md [hunk 2]
```diff
@@ -370,7 +370,7 @@ assertEqual(queue.complete(logScript).length, 1);
 
 Returns an array of job ids of jobs in the given queue with the status
 `"progress"`, optionally filtered by the given job type.
-The jobs will be looked up in the specified queue in the current database.
+The jobs is looked up in the specified queue in the current database.
```
Comment (nerpaula) [inline 3.10/foxx-reference-modules-queues.md:373]:
```suggestion
The job is looked up in the specified queue in the current database.
```

File: 3.10/foxx-reference-modules-queues.md [hunk 3]
```diff
@@ -392,7 +392,7 @@ The jobs will be looked up in the specified queue in the current database.
 
 Returns an array of job ids of jobs in the given queue with the status
 `"complete"`, optionally filtered by the given job type.
-The jobs will be looked up in the specified queue in the current database.
+The jobs is looked up in the specified queue in the current database.
```
Comment (nerpaula) [inline 3.10/foxx-reference-modules-queues.md:395]:
```suggestion
The job is looked up in the specified queue in the current database.
```

File: 3.10/foxx-reference-modules-queues.md [hunk 4]
```diff
@@ -436,7 +436,7 @@ The jobs will be looked up in the specified queue in the current database.
 
 Returns an array of job ids of all jobs in the given queue,
 optionally filtered by the given job type.
-The jobs will be looked up in the specified queue in the current database.
+The jobs is looked up in the specified queue in the current database.
```
Comment (nerpaula) [inline 3.10/foxx-reference-modules-queues.md:439]:
```suggestion
The job is looked up in the specified queue in the current database.
```

File: 3.10/programs-arangod-ldap.md [hunk 5]
```diff
@@ -95,23 +95,27 @@ other configuration options. For details see Section "LDAP URLs" below.
...
+```
+--ldap.enabled=true \
+--ldap.server=ldap.arangodb.com \
+--ldap.basedn=dc=arangodb,dc=com \
+--ldap.binddn=uid=arangoadmin,dc=arangodb,dc=com \
+--ldap.bindpasswd=supersecretpassword
+```
 
-The connection is identical but the searches will be executed with the
+The connection is identical but the searches is executed with the
```
Comment (nerpaula) [inline 3.10/programs-arangod-ldap.md:118]:
```suggestion
The connection is identical but the searches are executed with the
```

File: 3.10/programs-arangodump-examples.md [hunk 6]
```diff
@@ -73,47 +79,51 @@ arangodump \
...
 For example, to only dump structural information of all collections (including system
 collections), use:
 
-    arangodump --dump-data false --include-system-collections true --output-directory "dump"
+```
+arangodump --dump-data false --include-system-collections true --output-directory "dump"
+```
 
-To restrict the dump to just specific collections, there is is the *--collection* option.
+To restrict the dump to just specific collections, there is is the `--collection` option.
```
Comment (nerpaula) [inline 3.10/programs-arangodump-examples.md:99]:
```suggestion
To restrict the dump to just specific collections, use the `--collection` option.
```

File: 3.10/programs-arangoimport-details.md [hunk 7]
```diff
@@ -83,65 +85,65 @@ Importing the following document will then create an edge between *users/1234* a
...
   file and leaving all other attributes untouched.
 
-  The values of system attributes *_id*, *_key*, and *_rev* cannot be
+  The values of system attributes `_id`, `_key`, and `_rev` cannot be
   updated or replaced in existing documents.
 
-- *replace*: each document present in the import file that is also present in the target
-  collection already will be replace by arangoimport. *replace* will replace the existing
+- `replace`: each document present in the import file that is also present in the target
+  collection already is replace by arangoimport. `replace` replaces the existing
```
Comment (nerpaula) [inline 3.10/programs-arangoimport-details.md:114]:
```suggestion
  collection already is replaced by arangoimport. `replace` replaces the existing
```

File: 3.10/programs-arangoimport-details.md [hunk 8]
```diff
@@ -83,65 +85,65 @@ Importing the following document will then create an edge between *users/1234* a
...
+arangoimport returns the number of ignored documents in the `ignored` return value.
+When set to another value, `ignored` is always zero.
 
 It is possible to perform a combination of inserts and updates/replaces with a single
-arangoimport run. When *--on-duplicate* is set to *update* or *replace*, all documents present
-in the import file will be inserted into the target collection provided they are valid
-and do not already exist with the specified *_key*. Documents that are already present
-in the target collection (identified by *_key* attribute) will instead be updated/replaced.
+arangoimport run. When `--on-duplicate` is set to `update` or `replace`, all documents present
+in the import file is inserted into the target collection provided they are valid
```
Comment (nerpaula) [inline 3.10/programs-arangoimport-details.md:132]:
```suggestion
in the import file are inserted into the target collection provided they are valid
```

File: 3.11/foxx-reference-modules-queues.md [hunk 9]
```diff
@@ -294,9 +294,9 @@ queue.push(
 
 Creates a proxy object representing a job with the given job id.
 
-The job will be looked up in the specified queue in the current database.
+The job is looked up in the specified queue in the current database.
 
-Returns the job for the given jobId. Properties of the job object will be
+Returns the job for the given jobId. Properties of the job object are
 fetched whenever they are referenced and can not be modified.
```
Comment (nerpaula) [inline 3.11/foxx-reference-modules-queues.md:300]:
```suggestion
fetched whenever they are referenced and cannot be modified.
```

File: 3.11/foxx-reference-modules-queues.md [hunk 10]
```diff
@@ -370,7 +370,7 @@ assertEqual(queue.complete(logScript).length, 1);
 
 Returns an array of job ids of jobs in the given queue with the status
 `"progress"`, optionally filtered by the given job type.
-The jobs will be looked up in the specified queue in the current database.
+The jobs is looked up in the specified queue in the current database.
```
Comment (nerpaula) [inline 3.11/foxx-reference-modules-queues.md:373]:
```suggestion
The job is looked up in the specified queue in the current database.
```

File: 3.11/foxx-reference-modules-queues.md [hunk 11]
```diff
@@ -392,7 +392,7 @@ The jobs will be looked up in the specified queue in the current database.
 
 Returns an array of job ids of jobs in the given queue with the status
 `"complete"`, optionally filtered by the given job type.
-The jobs will be looked up in the specified queue in the current database.
+The jobs is looked up in the specified queue in the current database.
```
Comment (nerpaula) [inline 3.11/foxx-reference-modules-queues.md:395]:
```suggestion
The job is looked up in the specified queue in the current database.
```

File: 3.11/foxx-reference-modules-queues.md [hunk 12]
```diff
@@ -414,7 +414,7 @@ The jobs will be looked up in the specified queue in the current database.
 
 Returns an array of job ids of jobs in the given queue with the status
 `"failed"`, optionally filtered by the given job type.
-The jobs will be looked up in the specified queue in the current database.
+The jobs is looked up in the specified queue in the current database.
```
Comment (nerpaula) [inline 3.11/foxx-reference-modules-queues.md:417]:
```suggestion
The job is looked up in the specified queue in the current database.
```

File: 3.11/foxx-reference-modules-queues.md [hunk 13]
```diff
@@ -436,7 +436,7 @@ The jobs will be looked up in the specified queue in the current database.
 
 Returns an array of job ids of all jobs in the given queue,
 optionally filtered by the given job type.
-The jobs will be looked up in the specified queue in the current database.
+The jobs is looked up in the specified queue in the current database.
```
Comment (nerpaula) [inline 3.11/foxx-reference-modules-queues.md:439]:
```suggestion
The job is looked up in the specified queue in the current database.
```

File: 3.11/programs-arangod-ldap.md [hunk 14]
```diff
@@ -95,23 +95,27 @@ other configuration options. For details see Section "LDAP URLs" below.
...
+```
+--ldap.enabled=true \
+--ldap.server=ldap.arangodb.com \
+--ldap.basedn=dc=arangodb,dc=com \
+--ldap.binddn=uid=arangoadmin,dc=arangodb,dc=com \
+--ldap.bindpasswd=supersecretpassword
+```
 
-The connection is identical but the searches will be executed with the
+The connection is identical but the searches is executed with the
```
Comment (nerpaula) [inline 3.11/programs-arangod-ldap.md:118]:
```suggestion
The connection is identical but the searches are executed with the
```

File: 3.11/programs-arangodump-examples.md [hunk 15]
```diff
@@ -73,47 +79,51 @@ arangodump \
...
 For example, to only dump structural information of all collections (including system
 collections), use:
 
-    arangodump --dump-data false --include-system-collections true --output-directory "dump"
+```
+arangodump --dump-data false --include-system-collections true --output-directory "dump"
+```
 
-To restrict the dump to just specific collections, there is is the *--collection* option.
+To restrict the dump to just specific collections, there is is the `--collection` option.
```
Comment (nerpaula) [inline 3.11/programs-arangodump-examples.md:99]:
```suggestion
To restrict the dump to just specific collections, use the `--collection` option.
```

File: 3.11/programs-arangoimport-details.md [hunk 16]
```diff
@@ -83,65 +85,65 @@ Importing the following document will then create an edge between *users/1234* a
...
   file and leaving all other attributes untouched.
 
-  The values of system attributes *_id*, *_key*, and *_rev* cannot be
+  The values of system attributes `_id`, `_key`, and `_rev` cannot be
   updated or replaced in existing documents.
 
-- *replace*: each document present in the import file that is also present in the target
-  collection already will be replace by arangoimport. *replace* will replace the existing
+- `replace`: each document present in the import file that is also present in the target
+  collection already is replace by arangoimport. `replace` replaces the existing
```
Comment (nerpaula) [inline 3.11/programs-arangoimport-details.md:114]:
```suggestion
  collection already is replaced by arangoimport. `replace` replaces the existing
```

File: 3.11/programs-arangoimport-details.md [hunk 17]
```diff
@@ -83,65 +85,65 @@ Importing the following document will then create an edge between *users/1234* a
...
+arangoimport returns the number of ignored documents in the `ignored` return value.
+When set to another value, `ignored` is always zero.
 
 It is possible to perform a combination of inserts and updates/replaces with a single
-arangoimport run. When *--on-duplicate* is set to *update* or *replace*, all documents present
-in the import file will be inserted into the target collection provided they are valid
-and do not already exist with the specified *_key*. Documents that are already present
-in the target collection (identified by *_key* attribute) will instead be updated/replaced.
+arangoimport run. When `--on-duplicate` is set to `update` or `replace`, all documents present
+in the import file is inserted into the target collection provided they are valid
```
Comment (nerpaula) [inline 3.11/programs-arangoimport-details.md:132]:
```suggestion
in the import file are inserted into the target collection provided they are valid
```

**Review by nerpaula — COMMENTED** [review]
Found some other small typos, please see my suggestions. Other than that, LGTM.

File: 3.10/foxx-reference-modules-queues.md (hunk 2 above)
Comment (Simran-B) [reply 3.10/foxx-reference-modules-queues.md:373]: I think I didn't read this carefully enough and it should be `The jobs are looked up`

**Review by nerpaula — APPROVED** [review]

**Pull request merged**